main.pm

```diff
@@ -356,7 +356,7 @@ sub load_inst_tests() {
         loadtest "installation/upgrade_select.pm";
     }
     if (!get_var('LIVECD')) {
-        if (get_var('SCC_REGISTER') =~ m/installation/) {
+        if (get_var('SCC_REGISTER', 'installation') =~ m/installation/) {
```

Comment:
this will not run scc_registration.pm if SCC_REGISTER is not defined, mostly it is not defined.